ruleset.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<ruleset name="Google-Cloud-PHP-PSR2">
+  <rule ref="PSR2">
+    <!--
+      Exclude this sniff because we override methods with underscores in some classes
+    -->
+    <exclude name="PSR2.Methods.MethodDeclaration.Underscore"/>
+  </rule>
+  <exclude-pattern>tests/*</exclude-pattern>
```

Comment:
This configuration excludes all existing code, so it's currently a no-op. Was this intentional? 

(We could merge this change now and change the `exclude-pattern` definitions here in a follow-up change and run it across the existing code base. That might be something that's best done on our end though.)

I agree that there is value in providing a configuration here that is tailored to this project.

Reply:
We will lint the tests? My bad, I am not used to linting test files. I will do it in sperarate PR.

Reply:
I removed the exclusion of tests directory. All of tests did not violate `PSR-2`, so I won't be creating a new PR for it.

![image](https://user-images.githubusercontent.com/2193300/130245129-7770ddc2-9247-4285-9bbd-b07bc744de9c.png)